crypto/ec/curve448/Makefile

```diff
@@ -0,0 +1,38 @@
+all: libcurve448.so.1 curve448_test
```

Comment:
Is this file going to go away?

Reply:
Yes - I was using it temporarily before I had integrated the code into our build system.